### ISSUE TITLE
how to use image embeddings in layoutLM?

### ISSUE THREAD

**Issue by zhangliang-04** [BordiaS/layoutlm#3]
Hi,
I am confused about the image embeddings in the paper
1. Does the pre-training stage only use position embeddings and text embeddings, but not image embeddings? Since in Figure 2, image embeddings are only fed into the downstream block. If so, why the pre-training task called Masked _**Visual**_-Language Model? Since there is no vision-related input to the model during training.
2. I cannot find codes related to the image embeddings in this repo, where do you leverage it in the downstream task?
